### ISSUE TITLE
Missing method map for CreativeCommonsLicenseService

### ISSUE THREAD

**Issue by jcoyne** [sul-dlss/dor-services#465]
```
#<ActionView::Template::Error: undefined method `map' for Dor::CreativeCommonsLicenseService:Class>
/home/travis/build/sul-dlss/argo/vendor/bundle/ruby/2.5.0/gems/activesupport-5.2.2/lib/active_support/deprecation/proxy_wrappers.rb:26:in `method_missing'
/home/travis/build/sul-dlss/argo/app/forms/apo_form.rb:135:in `options_for_use_license_type'
/home/travis/build/sul-dlss/argo/app/forms/apo_form.rb:124:in `license_options'
```

**Comment by justinlittman** [comment]
Fixed by https://github.com/sul-dlss/dor-services/pull/466.

**Comment by jcoyne** [comment]
Fixed in #466
